Source/missiles.cpp

```diff
@@ -4464,54 +4464,27 @@ void __fastcall MI_Infra(int i)
 
 void __fastcall MI_Apoca(int i)
 {
-	int v1;        // esi
-	int v2;        // edi
-	signed int v3; // eax
-	int v4;        // ecx
-	int v5;        // ebx
-	int id;        // [esp+8h] [ebp-8h]
-	int v7;        // [esp+Ch] [ebp-4h]
+	int j, k, id, exit;

```

Comment:
if scalpel says `unsigned char` about something that is an `int`, it was probably a `BOOL`.
```suggestion
	int j, k, id;
	BOOL exit;
```